src/components/Drawer.js

```diff
@@ -164,8 +159,8 @@ const Drawer = React.createClass({
   },
 
   styles () {
-    return {
-      component: {
+    return StyleSheet.create({
```

Comment:
If we go this route I would rather pass in props as arguments:

```
//function execution
const styles = this.styles(this.props.style, this.props.contentStyle, this.props.headerStyle);

//function definition
styles (style, contentStyle, headerStyle) {
  component: Object.assign({}, {
  }, style),
//etc
}
```

Just personal preference and making functions take in arguments instead of referencing `this` properties often makes them easier to test if we ever needed / wanted to.